index.bs

```diff
@@ -459,6 +523,18 @@ dictionary IntersectionObserverInit {
 			"-10px 5px 8px"      // top = -10px, right & left = 5px, bottom = 8px
 			"-10px -5px 5px 8px" // top = -10px, right = -5px, bottom = 5px, left = 8px
 		</pre>
+	: <dfn>scrollMargin</dfn>
+	::
+		This is a string of 0-4 components,
+		each either an <a>absolute length</a> or a percentage.
+
+		<pre class="example" highlight=js>
```

Comment:
Can we make this match the syntax for `rootMargin`, and then just put `see rootMargin above` for the example?

Reply:
`scrollMargin` can be empty, so it could be a string of `0-4 components`, which is different from `rootMargin`

Reply:
Updated scrollMargin to match rootMargin